types/events.go

```diff
@@ -54,11 +55,17 @@ func RegisterEventDatas(cdc *amino.Codec) {
 
 type EventDataNewBlock struct {
 	Block *Block `json:"block"`
+
+	ResultBeginBlock abci.ResponseBeginBlock `json:"result_begin_block"`
```

Comment:
Should we treat these like we treat transactions, as independent events, with EventDataBeginBlock and EventDataEndBlock?

What are the use cases for getting just this data - is it more useful with the block itself or with other results tags from the block?